src/app/products/loan-products/wizard/loan-product-wizard.component.scss

```
@@ -0,0 +1,181 @@
+
+@media (width <= 768px) {
+  .wizard-shell__header,
+  .review-row {
+    flex-direction: column;
+    align-items: stretch;
+  }
+}
+
+// The reused Classic Charges step ships its own Previous/Next stepper row; the wizard shell already
+// renders unified step actions, so hide the embedded component's trailing nav to avoid duplicate buttons.
+.wizard-charges ::ng-deep .layout-row.margin-t {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)